### ISSUE TITLE
User menu outside of window

### ISSUE THREAD

**Issue by tanthammar** [stephenlake/nova-snowball#3]
fresh installation of Nova 2
After installing this theme, the user menu, top right is outside of view.
When I make the window smaller, so the responsiveness kicks in, the menu is visible again.

**Comment by stephenlake** [comment]
Version, browser, screenshot or PR? At least one or more would be highly appreciated.

**Comment by tanthammar** [comment]
Apologies.
Chrome 75.0.3770.142 (mac)
Nova 2.4.0
recording of my screen
https://www.dropbox.com/s/iutzmo3ro7ktwbo/Sk%C3%A4rminspelning%202019-10-12%20kl.%2010.22.39.mov?dl=0

**Comment by tanthammar** [comment]
This solves it.
fixed-header.css
row 18
```css
@media (max-width: 992px) {
  #nova .content {
    padding-top: 60px;
  }
  .h-header {
    padding-right: 2.2rem !important;
  }
}

@media (min-width: 992px) {
  .h-header {
    padding-right: 13rem !important;
  }
}
```

**Comment by tanthammar** [comment]
Created a PR on the other repo
https://github.com/tanthammar/nova-fixed-bars/pull/1

**Comment by XternalSoft** [comment]
@tanthammar , your PR make regression => Logo in header will disappear...

**Comment by stephenlake** [comment]
@beatwiz has submitted a PR fix for this. ~Need to do some testing before merging in.~ Merged and released as v1.1.2.

https://github.com/stephenlake/nova-fixed-bars/pull/6
